src-static/update_adm0_sf.R

```diff
@@ -232,14 +231,14 @@ download_adm0_sf <- function(iso3, update_azure = TRUE) {
 #' @param iso3 ISO3 code
 download_fieldmaps_sf <- function(iso3, layer = NULL) {
   iso3 <- tolower(iso3)
-  download_shapefile(
+  download_shapefile$download_shapefile(
     url = glue$glue("https://data.fieldmaps.io/cod/originals/{iso3}.gpkg.zip"),
-    data_source = "FieldMaps",
+    boundary_source = "OCHA, Fieldmaps",
```

Comment:
Think capitalizations should be: FieldMaps based on full attribution [here](https://fieldmaps.io/data)

_Attribution: FieldMaps, geoBoundaries, U.S. Department of State, U.S. Geological Survey_

Reply:
Aaah I see yes, strange that he refers to it as Fieldmaps elsewhere on the repo! Nice catch.

Reply:
And will switch to be `Fieldmaps, OCHA`